INSTALLATION.md

```diff
@@ -222,10 +222,10 @@ cd $GOPATH/src/github.com/alibaba/; git clone https://github.com/alibaba/pouch.g
 cd pouch; git checkout master
 ```
 
-Makefile target named `build` will compile the pouch and pouchd binaries in current work directory. Or you can just execute `make install` to build binaries and install them in destination directory (`/usr/local/bin` by default).
+Makefile target named `build` will compile the pouch and pouchd binaries in current work directory. You can execute `make build && make install` to build binaries and install them in destination directory (`/usr/local/bin` by default).
```

Comment:
build into `bin` directory actually 😄

Reply:
Yes, the `make build` would generate bin file into `bin` directory and `make install` would copy(or link?) this bin file to `/usr/local/bin` directory, is this pr anything wrong?

Reply:
I think we should update here `the pouch and pouchd binaries in current work directory` .

Reply:
OK, Got it. Thanks.